documentation/src/main/asciidoc/topics/marshalling.adoc

```diff
@@ -123,6 +93,73 @@ JBoss Marshalling is deprecated and planned for removal in a future version of {
 interface when persisting data unless you configure JBoss marshalling. However, this interface is also deprecated and planned for removal.
 ====
 
+=== Kryo Marshalling
+{brandname} provides a marshalling implementation that uses Kryo libraries.
+
+To use Kryo marshalling with {brandname}, do the following:
+
+. Add the Kryo marshaller dependency to your client `pom.xml`.
```

Comment:
This should be more generic now as it can be utilised on both the client, embdedded and server mode.